src/components/Charts/Charts.scss

```diff
@@ -1,8 +1,3 @@
 .pf-c-chart svg {
   overflow: visible !important;
-  height: 100% !important;
-}
-
-.pf-c-chart div {
-  height: 100% !important;
 }
```

Comment:
We can remove this file if you removed the import

Reply:
The Charts.scss is still used to overwrite the svg overflow (used for the axis).
I thought the same but I realized that it can't be removed yet.

Probably for a next round of refactoring.